gradle/src/main/groovy/com/flurry/android/symbols/NdkSymbolUpload.groovy

```diff
@@ -85,10 +81,17 @@ class NdkSymbolUpload {
         if (fileSet == null) {
             return;
         }
-        for(File dir: fileSet) {
-            if (dir.exists()) {
-                dir.eachDir { architecture ->
-                    architecture.eachFileMatch(FILES, ~/.*\.so$/, { processor(it) })
+        for(File file: fileSet) {
+            if (file.exists()) {
+                if (file.isDirectory()) {
+                    file.eachDir { architecture ->
+                        architecture.eachFileMatch(FILES, ~/.*\.so$/, { processor(it) })
```

Comment:
should we use `directory` instead of `architecture` here? Architecture might make sense in the context of a symbols package(?), but it doesn't really make sense in the context of the code...

Maybe `architectureDirectory` if you want to keep that around.

Reply:
updated to dir